docs/modules/ROOT/pages/configure-diagnostic-logging.adoc

```diff
@@ -0,0 +1,62 @@
+= Configure diagnostic logging
+:description: The Operator allows you to switch diagnostic logging on and off at runtime without having to restart the cluster. To prevent diagnostic logs from running indefinitely, you can also turn logging off automatically after a set period using the auto turn-off feature.
+:page-beta: true
+
+{description}
+
+This section explains how to configure diagnostics for Hazelcast clusters managed by the Hazelcast Platform Operator, including enabling diagnostics, forwarding diagnostics to your logging framework, setting auto turn-off to avoid disk overuse, and persisting diagnostics to a Persistent Volume Claim (PVC).
+
+== Enable diagnostics using the CRD
+
+The Operator exposes a diagnostics configuration you can set in the Hazelcast custom resource:
+
+* Fields include outputType (STDOUT, LOGGER, FILE), includeEpochTime, maxRolledFileSize, maxRolledFileCount, fileNamePrefix, pluginProperties, autoOffTimerInMinutes, and pvcName. See: https://docs.hazelcast.com/operator/latest-snapshot/api-ref#diagnostics[Diagnostics].
+* File-based rolling limits can also be controlled via DiagnosticsFileConfiguration (maxRolledFileSizeInMB, maxRolledFileCount, fileNamePrefix) where applicable. See: https://docs.hazelcast.com/operator/latest-snapshot/api-ref#diagnosticsfileconfiguration[DiagnosticsFileConfiguration].
+
+// question - is the following in the right place or should it appear later under the system properties section? Please advise
```

Comment:
It can be here as well. However, the default plugins are part of the core platform; therefore, I wouldn't mention them in here and only give a link to the platform documentation. Otherwise, in case of any changes on platform defaults, the Operator docs will be inconsistent with the core platoform one.